src/ripple/app/paths/cursor/DeliverNodeForward.cpp

```diff
@@ -35,7 +35,8 @@ TER PathCursor::deliverNodeForward (
     AccountID const& uInAccountID,    // --> Input owner's account.
     STAmount const& saInReq,        // --> Amount to deliver.
     STAmount& saInAct,              // <-- Amount delivered, this invocation.
-    STAmount& saInFees) const       // <-- Fees charged, this invocation.
+    STAmount& saInFees,             // <-- Fees charged, this invocation.
+    bool callerHasLiquidity) const
```

Comment:
What does `callerHasLiquidity` signify? I don't necessarily need a comment, but would like to make sure I understand the purpose of it, especially since we set it to `false` in all but one place, where it's calculated at runtime.